packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -30,6 +27,9 @@ const router = Router();
 router.post(
   "/directory/rebuild",
   asyncHandler(async (req: Request, res: Response) => {
+    if (Config.isSandbox()) return res.sendStatus(httpStatus.NOT_IMPLEMENTED);
```

Comment:
NABD, but this could become a local middleware, but let's get this out of the door now, can address this later.